control-plane/api/v1alpha1/mesh_webhook.go

```diff
@@ -59,7 +63,19 @@ func (v *MeshWebhook) Handle(ctx context.Context, req admission.Request) admissi
 		}
 	}
 
-	return admission.Allowed(fmt.Sprintf("valid %s request", mesh.KubeKind()))
+	return common.ValidateConfigEntry(ctx, req, v.Logger, v, &mesh, v.ConsulMeta)
```

Comment:
Double check my work but I think this was a bug that meant we weren't running any of the existing TLS validation.

Reply:
Excellent Catch!! thank you!!